package.js

```diff
@@ -102,37 +100,25 @@ Package.onTest(function (api) {
   ], ['client', 'server']);
 });
 
-// use meteor/fetch in tests only for NodeJS 8.11+ (Meteor 1.7+)
-function canRunTestsWithFetch () {
-  const nums = process.versions.node.split('.').map(Number);
-
-  const major = nums[0];
-  const minor = nums[1];
-
-  if (major < 8) return false;
-
-  if (major > 8) return true;
-
-  // major === 8 and ...
-  return minor >= 11;
-}
-
 function configurePackage (api, isTesting) {
-  api.versionsFrom('METEOR@1.4');
-  api.use('montiapm:meteorx@2.2.0', ['server']);
+  api.versionsFrom('METEOR@3.0-alpha.17');
+  api.use('montiapm:meteorx@2.3.1', ['server']);
   api.use('meteorhacks:zones@1.2.1', { weak: true });
   api.use('simple:json-routes@2.1.0', { weak: true });
-  api.use('zodern:meteor-package-versions@0.2.0');
-  api.use('zodern:types@1.0.9');
+
+  /**
+   * Uncomment once fibers is removed from the package.
+   */
+  // api.use('zodern:types@1.0.9');
```

Comment:
We should be able to re-add zodern:types

Reply:
It is still using fibers

<img width="635" alt="image" src="https://github.com/monti-apm/monti-apm-agent/assets/17411182/c1c73702-da72-46f8-b2a8-9d353a03f7a7">

Reply:
alpha 18 works